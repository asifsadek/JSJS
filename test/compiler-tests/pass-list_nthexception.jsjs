try List.nth(List.range(1, 10), 20) catch(e) {
  if e == "IndexOutOfBounds"
  then { print("n should be less than len"); 0; }
  else 0;
};
