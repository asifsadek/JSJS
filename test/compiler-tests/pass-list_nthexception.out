n should be less than len
